src/main/java/org/openpnp/vision/pipeline/CvPipeline.java

```diff
@@ -197,6 +198,14 @@ public void setNozzle(Nozzle nozzle) {
     public Nozzle getNozzle() {
         return nozzle;
     }
+	
+    public void setValue(double value) {
```

Comment:
> @cri-s I didn't ask you to remove the "closed loop and other functionality"
That is true,
.
> I'm asking you to remove setValue() which does not appear to be used. If
> it's not used, then what is the value of adding it? And if it *is* used,
> then commit the code that uses it. Right now none of the code you submitted
> uses it.
>
At the end this PR is for prerotate.
Removing this variable removes closed loop.feature, unfortunatly i
don't have code
to show the usage of this variable or what i have is totally untested
and too premature
for inclusion into this PR. If closed loop is requested in future, it
can be added later.
 
Cheers Cri.s